packages/server-nodejs/src/models/NodeDto.ts

```diff
@@ -1,11 +1,19 @@
 
-import { ArrayNotEmpty, IsUrl } from 'class-validator';
+import { z } from 'zod';
+
+export const schema = z.object({
```

Comment:
.object (and others) should start at a new line

Reply:
Done